subprojects/gradle/runner/service/src/main/kotlin/com/avito/runner/service/listener/TestMetricsSender.kt

```diff
@@ -0,0 +1,66 @@
+package com.avito.runner.service.listener
+
+import com.avito.android.stats.GaugeMetric
+import com.avito.android.stats.StatsDSender
+import com.avito.logger.LoggerFactory
+import com.avito.logger.create
+import com.avito.runner.service.model.TestCase
+import com.avito.runner.service.model.TestCaseRun
+import com.avito.runner.service.worker.device.Device
+import com.avito.time.TimeProvider
+
+class TestMetricsSender(
```

Comment:
```suggestion
internal class TestMetricsSender(
```
?

Reply:
yep, thanks; time to make module explicit )